apps/wakunode2/wakunode2.nim

```diff
@@ -34,23 +59,48 @@ when isMainModule:
   const versionString = "version / git commit hash: " & app.git_version
   let rng = crypto.newRng()
 
-  let confRes = WakuNodeConf.load(version=versionString)
+  let confRes = WakuNodeConf.load(version = versionString)
   if confRes.isErr():
-    error "failure while loading the configuration", error=confRes.error
+    error "failure while loading the configuration", error = confRes.error
     quit(QuitFailure)
 
-  let conf = confRes.get()
+  var conf = confRes.get()
+
+  # The Waku Network config (cluster-id=1)
+  if conf.clusterId == 1:
+    let twnClusterConf = ClusterConf.TheWakuNetworkConf()
+    if len(conf.shards) != 0:
+      conf.pubsubTopics = conf.shards.mapIt(twnClusterConf.pubsubTopics[it.uint16])
+    else:
+      conf.pubsubTopics = twnClusterConf.pubsubTopics
+
+    # Override configuration
+    conf.maxMessageSize = twnClusterConf.maxMessageSize
+    conf.clusterId = twnClusterConf.clusterId
+    conf.rlnRelay = twnClusterConf.rlnRelay
+    conf.rlnRelayEthContractAddress = twnClusterConf.rlnRelayEthContractAddress
+    conf.rlnRelayDynamic = twnClusterConf.rlnRelayDynamic
+    conf.rlnRelayBandwidthThreshold = twnClusterConf.rlnRelayBandwidthThreshold
+    conf.discv5Discovery = twnClusterConf.discv5Discovery
+    conf.discv5BootstrapNodes =
+      conf.discv5BootstrapNodes & twnClusterConf.discv5BootstrapNodes
```

Comment:
Is this overriding the configuration or just appending made on purpose?

Reply:
appending, just in case someone wants to add more bootstrap nodes on top of the existing ones.